<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>converter</class>
 <widget class="QMainWindow" name="converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>211</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>211</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(114, 159, 207);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="convert">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(164, 0, 0);</string>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
